### PR TITLE
[MWRAPPER-81] mvnw does not remove return chars from jvm.config

### PR DESCRIPTION
**Pull request by ArloL**
This PR adds `\r` to the character set to be replaced by a space when concatenating a file in mvnw. This is necessary on e.g. Windows.

Please let me know if I should provide more details like a project demonstrating the issue.

Following this checklist to help us incorporate your 
contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MWRAPPER) filed 
       for the change (usually before you start working on it).  Trivial changes like typos do not 
       require a JIRA issue.  Your pull request should address just this issue, without 
       pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[MWRAPPER-XXX] - Fixes bug in ApproximateQuantiles`,
       where you replace `MWRAPPER-XXX` with the appropriate JIRA issue. Best practice
       is to use the JIRA issue title in the pull request title and in the first line of the 
       commit message.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will 
       be performed on your pull request automatically.
 - [x] You have run the integration tests successfully (`mvn -Prun-its clean verify`).

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [ ] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

File: maven-wrapper-distribution/src/resources/mvnw [hunk 1]
```diff
@@ -160,7 +160,7 @@ find_maven_basedir() {
 # concatenates all lines of a file
 concat_lines() {
   if [ -f "$1" ]; then
-    echo "$(tr -s '\n' ' ' < "$1")"
+    echo "$(tr -s '\r\n' ' ' < "$1")"
```
Comment (bmarwell) [inline maven-wrapper-distribution/src/resources/mvnw:163]: Needs to be  rebased on top of https://github.com/apache/maven-wrapper/pull/79 later (removed that unnecessary echo call).

**Review by bmarwell — CHANGES_REQUESTED** [review]

File: maven-wrapper-distribution/src/resources/mvnw (hunk 1 above)
Comment (ArloL) [reply maven-wrapper-distribution/src/resources/mvnw]: I've updated the PR :)

**Comment by slawekjaranowski** [comment]
The same fix done by #86 
I missed that is the same ... 😄

**Comment by slawekjaranowski** [comment]
@ArloL by the way thanks

**Pull request closed without merging**